prompterator/constants.py

```diff
@@ -26,7 +47,9 @@ class ModelProperties(BaseModel):
 
     configurable_params: Dict[str, ConfigurableModelParameter] = {}
     non_configurable_params: Dict[str, Any] = {}
-
+    supports_structured_output: Optional[list[StructuredOutputImplementation]] = [
```

Comment:
Since it's a list and not a bool (as one would expect), how about
```suggestion
    supported_structured_output_implementations: Optional[list[StructuredOutputImplementation]] = [
```